src/components/Proposal/Create/PluginForms/CreatePluginManagerProposal.tsx

```diff
@@ -160,7 +160,7 @@ const votingParams: IGenesisProtocolFormValues = {
   votersReputationLossRatio: 4,
   minimumDaoBounty: 150,
   daoBountyConst: 10,
-  activationTime: moment().add(1, "day").format("YYYY-MM-DDTHH:mm"), //default date for the next day
+  activationTime: moment().add(10, "seconds").format("YYYY-MM-DDTHH:mm:ss"), // default date for next 10 seconds
```

Comment:
this is not good. 
we need to fix the test

Reply:
Fixed.